conf/systemd.service

```diff
@@ -50,8 +50,8 @@ SystemCallErrorNumber=EPERM
 #  https://bugs.archlinux.org/task/57721
 #  https://linux.m2osw.com/snapwebsites-postfixpostdrop18189-warning-mailqueueenter-create-file-maildrop25937318189-permission
 # Future wait for this ticket to be resolved: https://github.com/YunoHost/issues/issues/947
-SupplementaryGroups=postdrop
-ReadWritePaths=__FINALPATH__/config.ini __FINALPATH__/data/ /var/spool/postfix/maildrop/
+#SupplementaryGroups=postdrop
+#ReadWritePaths=__INSTALL_DIR__/config.ini __DATA_DIR__/ /var/spool/postfix/maildrop/
```

Comment:
Uncommenting this seems to fix the issue (at least the `__DATA_DIR__` part, I don't know about maildrop).
I have an issue with my Vm (500 errors on /yunohost/sso/) so I can't be sure at 100% but at least `service kresus start` works once this line is uncommented.

Reply:
Somehow it does not anymore…

Reply:
Fixed!